src/m365/spo/commands/page/page-control-set.mock.ts

```diff
@@ -0,0 +1,114 @@
+export const mockPage = {
```

Comment:
We should exclude this file from our coverage testing. It doesn't influence the coverage right now, but you never know 🙂